Source/KWStepper+Chaining.swift

```diff
@@ -3,6 +3,7 @@
 //  Created by Kyle Weiner on 1/22/17.
 //  https://github.com/kyleweiner/KWStepper
 //
+import Foundation
```

Comment:
Add line break here for consistent formatting with [other source files.](https://github.com/kyleweiner/KWStepper/blob/46bd5dbd4adb062e37992f62ad5f355528b4cbf1/Source/KWStepper.swift#L6)

```suggestion

import Foundation
```